Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
```

Comment:
This may sound stupid, but how does the ./vendor/bin/phpunit Tests/selenium/CheckCreateUser.php know to run the following functions:
generateTestData()
testAddPatient()
testFindPatient()

Does it basically run all public functions in order of placement?